module/actuation/FootController/data/config/webots/FootController.yaml

```diff
@@ -17,3 +19,17 @@ servo_gains:
   R_KNEE: 30
   R_ANKLE_PITCH: 30
   R_ANKLE_ROLL: 30
+
+balance:
+  enabled: false
```

Comment:
is there much of a reason to have this? It seems like all the modules (kick and walk) that use the balance control have their own configs that get passed into this module.